src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -26,4 +34,18 @@ static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> cac
  */
 static constexpr Property<bool, PropertyMutability::RW> exclusive_async_requests{"EXCLUSIVE_ASYNC_REQUESTS"};
 
+/**
+ * @brief the property for setting of required device for which config to be updated
+ * values: device id starts from "0" - first device, "1" - second device, etc
```

Comment:
I think plugins may have different devices naming convention, so this comment might be invalid in some cases

Reply:
> I think plugins may have different devices naming convention, so this comment might be invalid in some cases

Notice about another naming convention was added